src/hotspot/share/jvmci/jvmci_globals.hpp

```diff
@@ -127,9 +127,6 @@ class fileStream;
   product(bool, JVMCICountersExcludeCompiler, true, EXPERIMENTAL,           \
           "Exclude JVMCI compiler threads from benchmark counters")         \
                                                                             \
-  develop(bool, JVMCIUseFastLocking, true,                                  \
```

Comment:
You should just delete this option - JVMCI is clearly marked as experimental.
What you have in this PR is not proper deprecation. What if someone specifies `-XX:-JVMCIUseFastLocking`? They will get no warning or VM exit.

Reply:
Indeed. I have deleted the option.